lib/express.js

```diff
@@ -32,16 +57,22 @@ exports = module.exports = createApplication;
  * @api public
  */
 
-function createApplication() {
+function createApplication(opts) {
+  opts = opts || {}
```

Comment:
Probably don't want to reassign values linked to arguments, since it silently alters `arguments[0]`. Usually I would just name the argument `options` and this line as `var opts = options || {}`
```
$ node -pe '(function(foo){foo=foo||{};return arguments[0] === null}(null))'
false
```